src/adzerk/boot_reload/server.clj

```diff
@@ -13,24 +13,30 @@
 (defn set-options [opts]
   (reset! options opts))
 
-(defn web-path [protocol rel-path tgt-path asset-path]
-  (if-not (= "file:" protocol)
-    (str "/"
-      (if asset-path
-        (string/replace rel-path (re-pattern (str "^" (string/replace asset-path #"^/" "") "/")) "")
-        rel-path))
-    (.getCanonicalPath (io/file tgt-path rel-path))))
+(defn web-path
+  ([rel-path] (web-path {} rel-path))
+  ([opts rel-path]
+   (let [{:keys [protocol target-path asset-path cljs-asset-path]} opts]
+     (if (= "file:" protocol)
+       (.getCanonicalPath (io/file target-path rel-path))
+       (str
+        (if cljs-asset-path (str cljs-asset-path "/") "")
```

Comment:
@Deraen @esessoms I think this introduced a bug: paths are no longer starting with `/` breaking reloading from pushState locations. Should this line perhaps be:
```clojure
(str cljs-asset-path "/" ...)
```
So that the slash is always present?